contents/blog/the-posthog-array-1-34-0.md

```diff
@@ -0,0 +1,98 @@
+---
+date: 2022-03-24
+title: Array 1.34.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes", "Product updates"]
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.34.0 focuses on supporting collaboration. New features include a new default homepage for projects, smart names for all insights and an entire Data Management tool!
+---
+
+PostHog 1.34.0 focuses on supporting collaboration and helping you to find and share insights on PostHog. New features in this update include a new default homepage for projects, smart names for all insights and an entire Data Management tool!
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> Before upgrading, make sure to run all async migrations on your instance. This version will not run until async migrations are completed. Please check out the <a href="/docs/self-host/configure/async-migrations/overview" target="_blank">async migrations</a> docs for details.
+</blockquote>
+
+## PostHog 1.34.0 release notes
+
+> Wondering how to upgrade a self-hosted instance? Check out our [upgrade guide](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Project homepages](#new-homepage)
+- [Data Management](#new-data-management)
+- [Property group filtering](#new-property-group-filtering)
+- [Smart insight naming](#improvement-smart-insight-naming)
+- [Staff user controls](#new-staff-users)
+
+### New: Project homepages
+![PostHog - Project Homepages](../images/blog/array/1_34_0-homepage.png)
```

Comment:
This is the design, but the current implementation does not have these recently viewed/new panels. Could we do a product screenshot instead? @rcmarron

Reply:
Updated.